app/assets/stylesheets/mobile/header.css.scss

```diff
@@ -169,10 +163,20 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
-       -o-transform: translateX(-80%);
-      -ms-transform: translateX(-80%);
-     -moz-transform: translateX(-80%);
-  -webkit-transform: translateX(-80%);
+.draw {
```

Comment:
The selector here was too specific - due to the way the browser matches selectors, it actually hurts performance.